pages/learn/integration/integration-tutorials/index.js

```diff
@@ -0,0 +1,197 @@
+        <meta
+          name="description"
+          content="Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below."
+        />
+        <meta name="author" content="WSO2 LLC" />
+        <meta
+          name="keywords"
+          content="ballerina, learn, documentation, docs, programming language"
+        />
+        <link rel="shortcut icon" href="/img/favicon.ico" />
+        <title>Integration tutorials</title>
+
+        {/* FB */}
+        <meta property="og:type" content="article" />
+        <meta property="og:title" content="Ballerina - Learn" />
+        <meta
+          property="og:description"
+          content="Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below."
+        />
+        <meta
+          property="og:image"
+          itemProp="image"
+          content="https://ballerina.io/images/ballerina-generic-social-media-image-2023.png"
+        />
+
+        {/* LINKED IN */}
+        <meta property="og:title" content="Ballerina: Integration tutorials" />
+        <meta
+          property="og:image"
+          content="https://ballerina.io/images/ballerina-generic-social-media-image-2023.png"
+        />
+        <meta
+          property="og:description"
+          itemProp="image"
+          content="Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below."
+        />
+
+        {/* TWITTER */}
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="@ballerinalang" />
+        <meta name="twitter:creator" content="@ballerinalang" />
+        <meta name="twitter:title" content="Ballerina" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta
+          property="twitter:description"
+          content="Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below."
+        />
+        <meta
+          name="twitter:image"
+          content="https://ballerina.io/images/ballerina-generic-social-media-image-2023.png"
+        />
+        <meta
+          property="twitter:text:description"
+          content="Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below."
+        />
+        <meta
+          property="twitter:image"
+          content="https://ballerina.io/images/ballerina-generic-social-media-image-2023.png"
+        />
+      </Head>
+
+      <Layout>
+
+        <Col sm={12}>
+          <Row className="pageHeader pageContentRow llanding">
+
+            <Col xs={12}>
+              <Container>
+                <h1>Integration tutorials</h1>
+              </Container>
+            </Col>
+
+
```

Comment:
Two new lines. I'm not sure why we do this. I have seen this other places too (minor comment)

@sm1990/ @keizer619  Any Idea?